src/viur/shop/payment_providers/abstract.py

```diff
@@ -28,6 +28,16 @@ def title(self) -> translate:
         """Define the external title of the payment provider"""
         return translate(f"viur.shop.payment_provider.{self.name}", self.name)
 
+    @property
+    def description(self) -> translate:
+        """Define the description of the payment provider"""
+        return translate(f"viur.shop.payment_provider.{self.name}.descr", self.name)
+
+    @property
+    def image_path(self) -> str:
+        """Returns a path to an image for the payment provider"""
+        return ""
```

Comment:
That doesn't really make much sense in this form.

Can we also use or slightly misuse the translations for the paths?

Or make the whole thing setable as an instance attribute in the `__init__`? With a default value (`/static/images/shop/{self.name}.webp` or something)?

Furthermore i would prefer `None` as non existing value instead of an empty string.